projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -49,7 +49,7 @@ export class PageTimeRangePreferenceService {
         }
 
         // When FF is disabled
-        return () => this.getGlobalDefaultTimeRange();
+        return () => this.maybeGetCurrentTimeRange() ?? this.getGlobalDefaultTimeRange();
```

Comment:
nit - private method isn't really doing anything any more, can remove
```suggestion
        return () => this.timeRangeService.maybeGetCurrentTimeRange() ?? this.getGlobalDefaultTimeRange();
```

Reply:
I can address this in a follow up.